tests/DMSDocumentCartTest.php

```diff
@@ -76,6 +76,18 @@ public function testGetItems()
         $this->assertCount(1, $this->cart->getItems());
     }
 
+    public function testGetItemsAsArrayList()
+    {
+        $doc = $this->objFromFixture('DMSDocument', 'doc1');
+        /** @var DMSRequestItem $item */
```

Comment:
Yeah I don't think we need these - it's pretty obvious what class they are from the next line